src/checks/banFunctions.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+
+/**
+ * @description disallows use of a specific list of key words
+ * @param {string} [line] curr line being linted
+ * @return {boolean} true if translate3d is used, false if not
+ */
+var banFunctions = function( line ) {
+	var found = false
+	var bannedFunctions = this.config['bannedFunctions'] || []
```

Comment:
Bah, here it's called `bannedFunctions`, while in `checks` it's called `banFunctions`